.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,67 @@
+# A weekly workflow to run slow tests with MLFLOW_RUN_SLOW_TESTS environment variable set to true.
```

Comment:
Can we run this daily?

Reply:
I set this weekly because Docker building is not urgent comparing to other issues, like it doesn't cause any data/model loss to users. But I'm fine with setting daily if it doesn't cost much or increase load for our test infra (and make other tests wait)?

Reply:
> if it doesn't cost much or increase load for our test infra (and make other tests wait)?

Good point. It doesn't because this workflow will run on the mlflow-autmation fork.

Reply:
Oh I see, will increase it to daily before merging!